cui.cpp

```diff
@@ -215,6 +216,18 @@ void exit_ui ()
 
 void ui_tick ()
 {
+	fd_set fds;
+	
+	int maxfd = 0; // stdin
+
+	FD_ZERO(&fds);
+	FD_SET(0, &fds);
+	int status = select(maxfd + 1, &fds, 0, 0, 0);
```

Comment:
Aah now I understand better.

That could actually be a problem: catching packets is kind of time-critical, because for short-lived connections the connection might not be in /proc/net/tcp for long. When we process a message for a connection we haven't seen before, we check /proc/net/tcp again. 

So when processing messages is delayed, it would reduce the accuracy of nethogs' measurements.